### ISSUE TITLE
Accessibility: Screen Readers: When the web preview modal is brought into view, focus should be programmatically shifted to it.

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-calypso#2001]
This issue has been split out from: https://github.com/Automattic/wp-calypso/issues/697

When the web preview modal is brought into view, focus should be programmatically shifted to it. When it's closed, focus should shift back to the preview button. This way, a keyboard or screen reader user can use this feature and not lose their place. See one example of a [dialog modal](http://heydonworks.com/practical_aria_examples/#warning-dialog).

**Comment by marekhrabe** [comment]
I was doing some tests with this idea. I think the best solution would be to focus iframe contents on modal open, so users can scroll content by arrow keys. However, when we do that, we will lose the ability to listen for an escape key to close modal. It seems like we can have either one or the other when there are iframes in the way :(

**Comment by marekhrabe** [comment]
I found a solution that allows us to focus the iframe content, combine arrow keys to scroll + escape key to close and thus make preview amazingly seamless and accessible! Fix coming with #14330
